### PR TITLE
delete coffee icon fallback from backend

### LINKED ISSUE #915: Media Thumbnail controller fallback image (coffee icon) dont work correctly
bildschirmfoto 2015-03-16 um 11 57 27

Broken Images appears instead of fallback Icon

### PR DESCRIPTION
**Pull request by alexander-schranz**
__Tasks:__

- [ ] test coverage
- [ ] gather feedback for my changes
- [x] added changelog line
- [x] update fallback in husky datagrid https://github.com/massiveart/husky/pull/467
- [x] return 404 message

__Informations:__

| Q             | A
| ------------- | ---
| Tests pass?   | none
| Fixed tickets | none
| BC Breaks     | fixes #915
| Doc           | none

### CODE REVIEW

File: src/Sulu/Bundle/MediaBundle/Media/FormatManager/DefaultFormatManager.php [hunk 1]
```diff
@@ -191,21 +191,23 @@ public function returnImage($id, $formatName)
                 }
             } catch (MediaException $e) {
                 // return when available a file extension icon
-                list($image, $status, $imageExtension) = $this->returnFileExtensionIcon($formatName, $this->getRealFileExtension($fileName));
+                list($responseContent, $status, $imageExtension) = $this->returnFileExtensionIcon($formatName, $this->getRealFileExtension($fileName), $e);
             }
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Media/FormatManager/DefaultFormatManager.php]: do not nest try/catch blocks with the same Exception type ... this makes it really difficult to read ...

**Comment by wachterjohannes** [comment]
@alexander-schranz is this PR finished?

**Comment by alexander-schranz** [comment]
@wachterjohannes this PR need a husky update with the changes from this PR: https://github.com/massiveart/husky/pull/467

**Comment by danrot** [comment]
@wachterjohannes Did you merge this PR without waiting for your comments to be fixed?

**Pull request merged**